Documentation/Helm-Charts/operator-chart.gotmpl.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Ceph Operator Helm Chart
+---
+{{ template "generatedDocsWarning" . }}
```

Comment:
What it is auto-generating,is this file auto generated?

Reply:
This renders to a comment in the generated md file to warn users that the rendered doc is generated:

```markdown
<!---
Document is generated by `make helm-docs`. DO NOT EDIT.
Edit the corresponding *.gotmpl.md file instead
-->
```